packages/ppxlib_jane/ppxlib_jane.v0.17.2/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Jane Street developers"
+authors: ["Jane Street Group, LLC"]
+homepage: "https://github.com/janestreet/ppxlib_jane"
+bug-reports: "https://github.com/janestreet/ppxlib_jane/issues"
+dev-repo: "git+https://github.com/janestreet/ppxlib_jane.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/ppxlib_jane/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "5.3.0"}
```

Comment:
Is this correct? This works only with 5.3?

In this case we cannot test the revdeps, I suggest we just merge

Reply:
Yes, it's a fix for `ppxlib_jane.v0.17.0` failing on 5.3+.